var/spack/repos/builtin/packages/openmpi/btl_vader.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/opal/mca/btl/vader/btl_vader_fbox.h b/opal/mca/btl/vader/btl_vader_fbox.h
+index 17239ce8ef..ee5c913551 100644
+--- a/opal/mca/btl/vader/btl_vader_fbox.h
++++ b/opal/mca/btl/vader/btl_vader_fbox.h
+@@ -138,9 +138,6 @@ static inline bool mca_btl_vader_fbox_sendi (mca_btl_base_endpoint_t *ep, unsign
```

Comment:
ok, but the original patch seems to contain a bit more? Can this be ignored?
https://github.com/open-mpi/ompi/pull/5705/files

Reply:
The original patch combines two suggested patches from the original issue [1](https://github.com/open-mpi/ompi/issues/5375#issuecomment-420320320), [2](https://github.com/open-mpi/ompi/issues/5375#issuecomment-420415523). The comments mention the first one did not solve the issue, so I tried the second and it worked for the versions tested (2.0.5, 3.0.2, 3.1.2). I briefly tried building with the entire patch, but found that vader's source had diverged enough that it would require a separate patch for each minor release.